pkg/controller/wip/controller.go

```diff
@@ -552,11 +568,26 @@ func (c *controller) reconcileBinding(binding *v1alpha1.Binding) {
 	glog.V(4).Infof("Creating client for Broker %v, URL: %v", broker.Name, broker.Spec.URL)
 	brokerClient := c.brokerClientCreateFunc(broker.Name, broker.Spec.URL, username, password)
 
-	// TODO: parameters
+	parameters := make(map[string]interface{})
+	if len(binding.Spec.Parameters.Raw) > 0 {
+		err = yaml.Unmarshal([]byte(binding.Spec.Parameters.Raw), &parameters)
+		if err != nil {
```

Comment:
can you break the above 4 lines into a reusable function?

also, you can collapse the above 2 lines into:

```go
if err := yaml.Unmarshal(...); err != nil {
```